RelationalAI/Client.cs

```diff
@@ -454,13 +458,11 @@ public TransactionResult DeleteModel(string database, string engine, string name
         {
             var id = ExecuteAsync(database, engine, source, readOnly, inputs).Transaction.ID;
 
-            var transaction = GetTransaction(id).Transaction;
-
-            while (!(transaction.State.Equals("COMPLETED") || transaction.State.Equals("ABORTED")))
-            {
-                Thread.Sleep(2000);
-                transaction = GetTransaction(id).Transaction;
-            }
+            var transaction = Policy
+                .HandleResult<TransactionAsyncSingleResponse>(r =>
+                    !(r.Transaction.State.Equals("COMPLETED") || r.Transaction.State.Equals("ABORTED")))
+                .RetryForeverWithBoundedDelay()
```

Comment:
Is the default delay between retries 30 seconds ? If it's true this seems to be too much, we aim to have 1% wallclock time delay. Until we setup a policy for this let's use 1 sec instead.
Thanks for the exponential back-off :)